x-pack/plugins/cloud_security_posture/common/constants.ts

```diff
@@ -12,7 +12,7 @@ export const STATS_ROUTE_PATH = '/internal/cloud_security_posture/stats/{policy_
 export const BENCHMARKS_ROUTE_PATH = '/internal/cloud_security_posture/benchmarks';
 
 export const CLOUD_SECURITY_POSTURE_PACKAGE_NAME = 'cloud_security_posture';
-
+// TODO: REMOVE CSP_LATEST_FINDINGS_DATA_VIEW and replace it with LATEST_FINDINGS_INDEX_PATTERN
```

Comment:
can remove that now

Reply:
`CSP_LATEST_FINDINGS_DATA_VIEW` is used in multiple places. We added a todo comment for now.  I can create a follow PR to remove the instances where it's being used

Reply:
Got it, we can address it in follow-up tickets